debian/autobake-deb.sh

```diff
@@ -111,9 +119,10 @@ in
   "buster")
     disable_libfmt
     replace_uring_with_aio
-    ;&
+    ;;
   "bullseye")
     add_lsb_base_depends
+    change_compat_level 12
```

Comment:
The general pattern here is the config has the most recent version, and that early versions push this back.

So:

1. remove `change_compat_level 12` from existing locations.
1. add `change_compat_level 11` to `jammy` branch